changelog/issue-7965.md

```diff
@@ -0,0 +1,5 @@
+audience: general
+level: minor
```

Comment:
@matt-boris @petemoore what do you guys think about the level? This PR will drop `githubWebhookConsumer()` from client libs, so maybe it's a breaking one. But it was not supposed to be used by any clients, so likely `minor` is right? wdyt?